src/ApiCore/Transport/GrpcTransport.php

```diff
@@ -32,38 +32,73 @@
 
 namespace Google\ApiCore\Transport;
 
+use Exception;
 use Google\ApiCore\ApiException;
-use Google\ApiCore\AuthWrapper;
 use Google\ApiCore\BidiStream;
 use Google\ApiCore\Call;
 use Google\ApiCore\ClientStream;
+use Google\ApiCore\GapicHelpersTrait;
 use Google\ApiCore\ServerStream;
+use Google\ApiCore\ValidationException;
+use Google\ApiCore\ValidationTrait;
 use Google\Rpc\Code;
 use Grpc\BaseStub;
 use Grpc\Channel;
+use Grpc\ChannelCredentials;
 use GuzzleHttp\Promise\Promise;
+use Throwable;
 
 /**
  * A gRPC based transport implementation.
  */
 class GrpcTransport extends BaseStub implements TransportInterface
 {
+    use ValidationTrait;
+    use GapicHelpersTrait;
+
     /**
-     * @param string $host The domain name and port of the API remote host.
-     * @param array $stubOpts An array of options used when creating a BaseStub.
-     * @param Channel $channel An already instantiated channel to be used during
-     *        creation of the BaseStub.
+     * Builds a GrpcTransport.
+     *
+     * @param string $serviceAddress
+     *        The address of the API remote host, for example "example.googleapis.com. May also
+     *        include the port, for example "example.googleapis.com:443"
+     * @param array $config
+     *        Config options used to construct the gRPC transport. Supported options are 'stubOpts'
+     *        and 'channel'.
```

Comment:
Could we use a style like this for both transports? It makes understanding the expected types easier

```php
    /**
     * @param array $config {
     *    Config options used to construct the gRPC transport.
     *    
     *    @type array $stubOpts Description
     *    @type Channel $channel Description
     * } 
     */
```